pkgs/development/tools/djlint/default.nix

```diff
@@ -5,14 +5,14 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "djlint";
-  version = "1.32.1";
+  version = "1.34.1";
   format = "pyproject";
```

Comment:
Please consider to use `pyproject = true` instead of `format = "pyproject"`.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters